scripts/update_all.py

```diff
@@ -37,17 +39,17 @@
             if os.path.isfile(strParamsItem):
                 paramsAll+= ' -c '+strParamsItem
             else:
-                print('ERROR: Configuration file %s not found.', strParamsItem)
+                print(('ERROR: Configuration file %s not found.', strParamsItem))
```

Comment:
Why double brackets?

Reply:
Try it and see (in Python 2)...
```python
print “a”, “b”
print(“a”, “b”)
print((“a”, “b”))
```

The first and last print the same thing but the middle one does not.

Reply:
For me in the Python 2.7.14 the second and the last are the same 
```
>>> print("a", "b")
('a', 'b')
>>> print(("a", "b"))
('a', 'b')
```
But anyway, this row should be rather `print('ERROR: Configuration file %s not found.' % strParamsItem)`

Reply:
Do you have `from __future__ import print_function` at the top of the file?